trezorlib/tests/device_tests/test_msg_cardano_sign_transaction.py

```diff
@@ -150,3 +229,32 @@ def input_flow():
         )
         assert response.tx_hash.hex() == tx_hash
         assert response.tx_body.hex() == tx_body
+
+
+@pytest.mark.cardano
+@pytest.mark.skip_t1  # T1 support is not planned
+@setup_client()
+@pytest.mark.parametrize(
+    "protocol_magic,inputs,outputs,transactions,expected_error_message", INVALID_VECTORS
+)
+def test_cardano_sign_tx_validation(
+    client, protocol_magic, inputs, outputs, transactions, expected_error_message
+):
+    inputs = [cardano.create_input(i) for i in inputs]
+    outputs = [cardano.create_output(o) for o in outputs]
+
+    expected_responses = [
+        messages.CardanoTxRequest(tx_index=i) for i in range(len(transactions))
+    ]
+    expected_responses += [messages.Failure()]
+
+    with client:
+
```

Comment:
less whitespace please :))

Reply:
seems to be introduced by "make style" when I run it